make.inc

```diff
@@ -102,6 +102,11 @@ ARFLAGS  = $(ARFLAGS_$(arch))
 DEFINES += $(addprefix -D , $D)
 CLEANFILES += $(O) *.o *.a $(all_tests) $(bin_PROGRAMS) $(lib_name) $(so_lib_name) $(all_llvm_fuzz_tests)
 
+# add armv8 crc crypto option support
+ifeq ($(shell uname -m), aarch64)
+CFLAGS  += -march=armv8-a+crc+crypto
```

Comment:
If delete these lines, will cause the following error. this is the basic aarch64 CFLAGS option, all aarch64 should support it. for example, armv9 must be backward compatible. So it will not preclude other arm.
...
crc/aarch64/crc16_t10dif_pmull.S: Assembler messages:
crc/aarch64/crc16_t10dif_pmull.S:213: Error: selected processor does not support `pmull v_x0_h.1q,v_x0_h.1d,v_p4_h.1d'
crc/aarch64/crc16_t10dif_pmull.S:214: Error: selected processor does not support `pmull v_x0_l.1q,v_x0_l.1d,v_p4_l.1d'
crc/aarch64/crc16_t10dif_pmull.S:215: Error: selected processor does not support `pmull v_x1_h.1q,v_x1_h.1d,v_p4_h.1d'
crc/aarch64/crc16_t10dif_pmull.S:216: Error: selected processor does not support `pmull v_x1_l.1q,v_x1_l.1d,v_p4_l.1d'
crc/aarch64/crc16_t10dif_pmull.S:217: Error: selected processor does not support `pmull v_x2_h.1q,v_x2_h.1d,v_p4_h.1d'
crc/aarch64/crc16_t10dif_pmull.S:218: Error: selected processor does not support `pmull v_x2_l.1q,v_x2_l.1d,v_p4_l.1d'
crc/aarch64/crc16_t10dif_pmull.S:219: Error: selected processor does not support `pmull v_x3_h.1q,v_x3_h.1d,v_p4_h.1d'
crc/aarch64/crc16_t10dif_pmull.S:220: Error: selected processor does not support `pmull v_x3_l.1q,v_x3_l.1d,v_p4_l.1d'